src/hardware/meson.build

```diff
@@ -24,6 +24,7 @@ libhardware_sources = files(
     'adlib_gold.cpp',
     'cmos.cpp',
     'covox.cpp',
+    'disk_noise.cpp',
     'disney.cpp',
     'dma.cpp',
```

Comment:
Don't bother with this. Meson will go away in a month max. Please use CMake.

Reply:
I'd like to Meson working until we're ready to fully switch over to CMake meaning everything in #4196 has been fixed. Until then, it's what I'm using for bisections so I don't have to test whether some regression is actually just a CMake bug.

Reply:
Okay, we can keep it around for a bit longer, but then all our CI jobs use CMake already. It's a bit pointless IMO because you should really be using CMake for local dev too at this point.

At the very least, make sure you update the CMake build scripts as well @dividebysandwich.

Reply:
What if someone opens a regression ticket for ManyMouse or networking? I would have no way to bisect that as those features aren't working in CMake.

Also when we do add those features to CMake, if problems pop up we'd want a working Meson build to see if it's a build system issue or something else.

Reply:
Fair points. I'm just eager to kill it finally 😆  But we'll keep it around for a few more months then.